test/tests/testharness.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(t => {
+  assert_true(true);
+}, "test");
+
+promise_test(t => {
+  return Promise.resolve(() => {
```

Comment:
Passing a function to Promise.resolve() just resolves the promise with that function. Probably you should just not pass anything.